schema.sql

```diff
@@ -10,4 +10,23 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-       ADD COLUMN species VARCHAR(100); 
\ No newline at end of file
+       ADD COLUMN species VARCHAR(100); 
+
+CREATE TABLE owners (
+       id int NOT NULL GENERATED ALWAYS AS IDENTITY,
+       full_name varchar(100);
+       age int;
+   	CONSTRAINT owners_pk PRIMARY KEY (id)
+   );
+
+   CREATE TABLE species (
+       id int NOT NULL GENERATED ALWAYS AS IDENTITY,
+       name varchar(100);
+       PRIMARY KEY(id)
+   );
+
+   ALTER TABLE animals DROP COLUMN species;
+   ALTER TABLE animals ADD species_id int NULL;
+   ALTER TABLE animals ADD owner_id int NULL;
```

Comment:
**Changes Required**
- [ ] Column `species_id` and `owner_id` cannot be `NULL`. This will conflict with integral constraint since they will be used as `FOREIGN KEYS`. Let us please remove the NULL constraints.

Reply:
:+1: This is addressed :heavy_check_mark: